polygon-id/issuer/Dockerfile

```diff
@@ -1,26 +1,19 @@
-FROM golang:1.20.5 as base
+FROM golang:1.21 as base
 ARG VERSION
 WORKDIR /service
 ENV GOBIN /service/bin
-
-COPY ./go.mod ./
-COPY ./go.sum ./
-
-# Download dependencies and cache them
-RUN go mod download
```

Comment:
it was done like this to cache dependencies download, otherwise rebuild was very slow

Reply:
fixed